actix-rt/src/arbiter.rs

```diff
@@ -151,7 +159,7 @@ impl Arbiter {
         Arbiter::spawn(future::lazy(f))
     }
 
-    /// Send a future on the arbiter's thread and spawn.
+    /// Send a future to the Arbiter's thread, and spawn it.
```

Comment:
it's a detail, but I wouldn't use ", and". The comma shouldn't be there if you have an "and".